### PR TITLE
Validation answer + correction special symbols

### PR DESCRIPTION
**Pull request by n-faycel**

### CODE REVIEW

**Review by remi59800 — DISMISSED** [review]
okay

**Review by Gonaline — APPROVED** [review]

**Review by Gonaline — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me,

File: frontend/src/components/Buttons/Answer/index.jsx [hunk 1]
```diff
@@ -1,34 +1,37 @@
...
 import Style from "./style";
 
-export default function Answer({ data: { correctAnswer } }) {
+export default function Answer({ data: { correctAnswer, incorrectAnswers } }) {
+  const answers = [];
+  function arrayOfAnswers() {
+    for (let i = 0; i < incorrectAnswers.length; i += 1) {
+      answers.push(incorrectAnswers[i]);
+    }
+    answers.push(correctAnswer);
```
Comment (LoicBrassart) [inline frontend/src/components/Buttons/Answer/index.jsx]: Nope, use a state!

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
You should never display a value which is not a state

**Review by LoicBrassart — APPROVED** [review]
No moe Evil

**Pull request merged**